docs/source/examples.rst

```diff
@@ -336,18 +337,27 @@ sort_by_difficulty.py
             # It's very unlikely for the title property to be blank or missing.
             # This is mostly to satisfy type-checkers.
             current_title = sf.title or ""
+            current_titletranslit = sf.titletranslit or ""
 
             if args.remove:
-                # Look for a number in brackets at the start of the title
-                if current_title.startswith("["):
-                    open_bracket_index = current_title.find("[")
-                    close_bracket_index = current_title.find("]")
-                    bracketed_text = current_title[
-                        open_bracket_index + 1 : close_bracket_index
-                    ]
-                    if bracketed_text.isnumeric():
-                        # Remove the bracketed number from the title
-                        sf.title = current_title[close_bracket_index + 1 :].lstrip(" ")
+                def remove_starting_brackets(current_text: str) -> str:
+                """
+                If current_text has a bracketed number at the start of the text, remove it and return it
+                Otherwise, return current_text unchanged. 
+                """
+                # Look for a number in brackets at the start of the text
```

Comment:
These lines should be indented to match the `if current_text.startswith("["):` line below